app/controllers/subscribers.js

```diff
@@ -108,10 +108,13 @@ export default Controller.extend(PaginationMixin, {
 
         sortByColumn(column) {
             let table = this.get('table');
+            let valuePath = column.get('valuePath');
+
+            valuePath = valuePath === 'createdAtUTC' ? 'created_at' : valuePath.trim().underscore();
```

Comment:
Cool, all this bit of code is doing is setting the query params (`.../subscribers/?order=created_at&direction=asc`) - right now those query params are passed directly to the server so need to match the server-side column names.

In this case a little bit of string manipulation to remove the `UTC` part inline is fine, if the logic get more complex or needs to be repeated in many areas then we can look at modifying the adapter so our URLs can use the client's column names but the adapter converts them to server column names when making requests to the API.